stable/metrics-server/templates/metrics-server-deployment.yaml

```diff
@@ -18,11 +18,20 @@ spec:
       labels:
         app: {{ template "metrics-server.name" . }}
         release: {{ .Release.Name }}
+      {{- if .Values.podAnnotations }}
+      annotations:
```

Comment:
A small thing, but can you change this part to use `with` similar to the nodeSelector part further down in this file?

Reply:
Thanks for your comment! I fixed ```if``` to ```with``` 🙂in https://github.com/helm/charts/pull/12110/commits/c80e4a7e1dd51bbfb15fbdf3682fa61468255926